sql/moz-fx-data-shared-prod/monitoring_derived/bigquery_usage_v2/metadata.yaml

```diff
@@ -8,6 +8,11 @@ labels:
   schedule: daily
 scheduling:
   dag_name: bqetl_monitoring
+  depends_on:
+  - dag_name: bqetl_monitoring_hourly
+    task_id: monitoring_derived__jobs_by_organization__v1
+    # wait for run at 6:00 UTC
+    execution_delta: -4h
```

Comment:
This comment is helpful to indicate that waiting for a later run of the hourly job is intended behavior, but it'd also be helpful to indicate why you chose to do this (i.e. why not just rely on the run that happens at 02:00 UTC like it'd do by default without the `execution_delta` setting)?

Reply:
Sure I'll add more.  It's to get the jobs that start before midnight but take more than a couple hours to finish since some stats are only available for completed jobs